### ISSUE TITLE
开始时间问题

### ISSUE THREAD

**Issue by CPunisher** [roife/BUAAIcsGen#1]
我们不是2.24开学🐎
`termBeginDate = datetime.date(2020, 2, 23)`
我导入才发现所有课程都提前了一天

**Comment by GrapeLemonade** [comment]
Same problem, with ics output as follows:

```plain
BEGIN:VEVENT
DTSTAMP:20190902T123110Z
DTSTART;TZID=Asia/Shanghai:20200223T080000
DTEND;TZID=Asia/Shanghai:20200223T093500
RRULE:FREQ=WEEKLY;COUNT=16
SUMMARY:基础物理学（信息类）
LOCATION:主M404
DESCRIPTION:基础物理学（信息类） 庞丹阳，张玉洁[1-16]周主M404 第1，2节
BEGIN:VALARM
TRIGGER:-PT60M
REPEAT:1
DURATION:PT1M
END:VALARM
END:VEVENT
```

2020 Spring Semester starts on Feb. 24.

**Comment by roife** [comment]
fixed in `06d376f`
